src/parameters/TerminalVelocity.jl

```diff
@@ -163,6 +170,13 @@ function Chen2022VelTypeSnowIce(toml_dict::CP.AbstractTOMLDict)
         :Chen2022_table_B3_Es => :Es,
         :Chen2022_table_B3_Fs => :Fs,
         :Chen2022_table_B3_Gs => :Gs,
+        :Chen2022_table_B5_Al => :Al,
+        :Chen2022_table_B5_Al => :Bl,
+        :Chen2022_table_B5_Al => :Cl,
+        :Chen2022_table_B5_Al => :El,
+        :Chen2022_table_B5_Al => :Fl,
+        :Chen2022_table_B5_Al => :Gl,
+        :Chen2022_table_B5_Al => :Hl,
```

Comment:
```suggestion
        :Chen2022_table_B5_Al => :Al,
        :Chen2022_table_B5_Bl => :Bl,
        :Chen2022_table_B5_Cl => :Cl,
        :Chen2022_table_B5_El => :El,
        :Chen2022_table_B5_Fl => :Fl,
        :Chen2022_table_B5_Gl => :Gl,
        :Chen2022_table_B5_Hl => :Hl,
```
I think these need to be renamed! The error in the tests is pretty unclear.